src/m365/spo/commands/list/list-list.ts

```diff
@@ -36,7 +36,7 @@ class SpoListListCommand extends SpoCommand {
     }
 
     const requestOptions: any = {
-      url: `${args.options.webUrl}/_api/web/lists?$expand=RootFolder`,
+      url: `${args.options.webUrl}/_api/web/lists?$expand=RootFolder&$select=RootFolder/ServerRelativeUrl,*`,
```

Comment:
Does this change affect the shape of the data returned by the command? If so, then it's a breaking change that we can't just implement, even if it's blocking the command usage in some scenarios. For backwards compatibility, we'd need to introduce it behind a switch and then clean it up in the next major version.

Reply:
Good question, It does return the entire list object, but now I think of it, it does return less info on the RootFolder object

Reply:
In that case, we'd need to temporarily introduce an extra option that will allow you to run the command with limited permissions

Reply:
I'll update the code